File: engine/engine-model/src/main/scala/net/server/channel/packet/command/admin/ChangeMapPacket.scala
package net.server.channel.packet.command.admin

import net.server.MaplePacket

class ChangeMapPacket( private var _mode: Byte,  private var _victim: String,  private var _mapId: Int) extends BaseAdminCommandPacket(_mode = _mode) {
     def victim: String = _victim
     def mapId: Int = _mapId
}

File: engine/engine-model/src/main/java/net/server/channel/packet/command/admin/ChangeMapPacket.java
package net.server.channel.packet.command.admin;

public class ChangeMapPacket extends BaseAdminCommandPacket {
   private final String victim;

   private final Integer mapId;

   public ChangeMapPacket(Byte mode, String victim, Integer mapId) {
      super(mode);
      this.victim = victim;
      this.mapId = mapId;
   }

   public String victim() {
      return victim;
   }

   public Integer mapId() {
      return mapId;
   }
}
